src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -183,6 +194,10 @@ into your targets's build settings > General > Frameworks, Libraries, and Embedd
 
 {% include app-figure.md image="development/add-to-app/ios/project-setup/embed-xcode.png" alt="Embed frameworks in Xcode" %}
 
+In the target's build settings, add `$(PROJECT_DIR)/Flutter/Release/` to your Framework Search Paths.
+
+{% include app-figure.md image="development/add-to-app/ios/project-setup/framework-search-paths.png" alt="Update Framework Search Paths in Xcode" %}
```

Comment:
the canvas is 850x400 but looks really pixelated when I downloaded the raw PNG. Maybe something happened to it.

Reply:
actually everything looks a bit messed up on my machine so it's likely just me

Reply:
It was pixely, I tried it again.